uwblueprint/jump-math
# feat: update UserService with new fields

## Notion ticket link

[Update userService to store grade selection and currently using fields](https://www.notion.so/uwblueprintexecs/Update-userService-to-store-grade-selection-and-currently-using-fields-db2d028ad4c149feb34d6eb103ca1348)



## Implementation description
* Updated `User` Model to store new fields, UserService implementation, GraphQL types, and tests
* Updated `registerTeacher` resolver to pass new fields to `createUser`



## Steps to test
1. Run yarn `test`
2. Test User service API calls via GraphQL Playground
3. Make test users and check records in Mongo DB
<img width="332" alt="image" src="https://user-images.githubusercontent.com/69697180/210935239-be1f0ffa-6bcc-4280-8155-42628eba9d98.png">

when both fields are `undefined` (default)
<img width="313" alt="image" src="https://user-images.githubusercontent.com/69697180/209459106-e314aef8-b517-4355-949d-18abf570012c.png">



## What should reviewers focus on?
* 


## Checklist
